contact.html

```diff
@@ -11,11 +11,20 @@
       <h1>Contact Us</h1>
       <p>Whether you just have a couple questions or if you’re ready to sign up and start using Kubo Health, let us know how we can help.</p>
     </div>
-    <form method="post" accept-charset="UTF-8" action="http://www.formfor.com/submissions/Xa8woRRymS9urHkCuGRyQy8P" target="_blank">
+    <!-- <form method="post" accept-charset="UTF-8" action="http://www.formfor.com/submissions/Xa8woRRymS9urHkCuGRyQy8P" target="_blank">
       <input type="text" name="name" placeholder="Name">
       <input type="email" id="email" name="email" placeholder="Email" oninput="checkValidEmail()">
       <textarea name="comment" placeholder="How can we help you?"></textarea>
       <input type="submit" id="submit" name="commit" value="Send Message" class="button-secondary" disabled>
+    </form> -->
+    <form method="post" id="formFor" accept-charset="UTF-8" action="http://www.formfor.com/submissions/Xa8woRRymS9urHkCuGRyQy8P" target="_blank">
+      <input type="text" name="name" placeholder="Name">
+      <input type="text" id="email" name="email" placeholder="Email">
```

Comment:
@paulgonz6 In the old form version on FF, we had this:
`<input type="email" id="email" name="email" placeholder="Email" oninput="checkValidEmail()">`. Is the `oninput` concept removed from the new form?